tests/integration/conftest.py

```diff
@@ -97,7 +97,10 @@ def token(pytestconfig: pytest.Config) -> str:
     token = pytestconfig.getoption("--token")
     assert token, "Please specify the --token command line option"
     secondary_tokens: str = pytestconfig.getoption("--secondary-tokens", default="")
-    tokens = secondary_tokens.split(",").append(token)
+    tokens = [
+        token.strip() for token in (secondary_tokens.split(",") if secondary_tokens != "" else [])
```

Comment:
"".split(",") will already return an empty array, so no need for an if else here

Reply:
`"".split(",")` returns a list with an item, i.e. [""] unfortunately :/

Reply:
Oops, you're right. My bad